README.md

```diff
@@ -35,3 +35,12 @@ Make sure the service is up first using `docker-compose up` then run:
 ```sh
 docker-compose exec core python manage.py test
 ```
+
+to run unit tests. Or, to load cancer-static data:
+
+```
+docker-compose exec core bash
+python manage.py acquiredata 
```

Comment:
My rationale was to be consistent with Django's existing management commands, like `runserver` and `makemigrations`, which are not `run_server` or `make_migrations`. Seems to be a tradeoff between a consistent use of underscores and readability. If anyone has strong feelings, I can change it.